### PR TITLE
*: Reuse operation name constants for consistency

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

*  [x] The Changelog entry added.

## Verification

* `make test-local`

### CODE REVIEW

File: pkg/store/cache/caching_bucket.go [hunk 1]
```diff
@@ -29,12 +29,6 @@ import (
 const (
 	originCache  = "cache"
 	originBucket = "bucket"
-
-	opGet        = "get"
-	opGetRange   = "getrange"
```
Comment (kakkoyun) [inline pkg/store/cache/caching_bucket.go:34]: This was actually mismatching what we have already for bucket.

Comment (pracucci) [reply pkg/store/cache/caching_bucket.go:34]: Doh! I think we should mention this change in the CHANGELOG

Comment (bwplotka) [reply pkg/store/cache/caching_bucket.go:34]: Any mixin / alert we could change?

**Comment by kakkoyun** [comment]
@bwplotka Mixin seems ok. We don't have any usage. So merging it.

File: pkg/store/cache/caching_bucket.go (hunk 1 above)
Comment (kakkoyun) [reply pkg/store/cache/caching_bucket.go:34]: Nothing, I could find.

**Pull request merged**